Template and VF Components/QueryComponent.component

```diff
@@ -0,0 +1,28 @@
+<apex:component controller="QueryComponentController">
+    <apex:attribute name="inputValue" description="Attribute on the component." type="String" required="required" assignTo="{!inputQuery}"/>
```

Comment:
the `description` of the apex:attribute should be meaningful like
`The query to fetch`, `The query to execute`

Reply:
changed it.